src/lib/crypto/kimchi_bindings/stubs/src/plonk_verifier_index.rs

```diff
@@ -70,13 +70,60 @@ where
     }
 }
 
+#[derive(ocaml::IntoValue, ocaml::FromValue, ocaml_gen::Struct)]
+pub struct CamlLookupInfo {
```

Comment:
why not use LookupInfo directly? I'm guessing the only things that didn't work out were the `usize`? In which case we can just support them in ocaml-gen or use u32 instead?

Reply:
> also all of these types are usually defined in the proof-system crate

I think you started doing that, but there are still several that live here. We should migrate them over, but I'd rather not do it in this PR.

Reply:
> I think you started doing that

this predates me o.o